docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -147,59 +150,65 @@ All fields in the ``DOCUMENTATION`` block are lower-case. All fields are require
   * A detailed description (generally two or more sentences).
   * Must be written in full sentences, i.e. with capital letters and periods/full stops.
   * Shouldn't mention the module name.
+  * Make use of multiple entries rather than using one long paragraph.
+  * Don't quote complete values unless it is required by YAML.
 
 :version_added:
 
   * The version of Ansible when the module was added.
-  * This is a string, and not a float, i.e. ``version_added: "2.1"``
+  * This is a string, and not a float, i.e. ``version_added: '2.1'``
 
 :author:
 
   * Name of the module author in the form ``First Last (@GitHubID)``.
   * Use a multi-line list if there is more than one author.
+  * Don't use quotes as it should not be required by YAML.
 
 :deprecated:
 
   * Marks modules that will be removed in future releases. See also :ref:`module_lifecycle`.
 
 :options:
 
-  * If the module has no options (for example, it's a ``_facts`` module), all you need is one line: ``options: {}``.
-  * If your module has options (in other words, accepts arguments), each option should be documented thoroughly. For each module argument/option, include:
+  * Options are usually called `parameters` and sometimes arguments. We will be using `parameters` for the remainder of the documentation.
```

Comment:
The documentation field is called `options`, not `parameters` - so it's a bit confusing to say "create a field called `options`" and then document it as if it were called `parameters`. I like the move toward consistency, for sure, but describing a field with one name using a different name seems like a step backward.

Reply:
Sure, but we use 'parameter' almost everywhere else (including the module documentation), so why complicate things in the documentation, just because it is called `options` in the module docs spec? I am all for renaming this part in Ansible too, but that won't happen in this PR.

Reply:
PS We are already having 3 names (option, argument, parameter) for the same thing, so I don't think my changes is *taking a step backward*. At least we are clarifying that it is the same thing, and we standardize on a single name.